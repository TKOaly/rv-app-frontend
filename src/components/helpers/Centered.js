import React from 'react';
import './styles/Centered.css';

const Centered = (props) => (
    <div className="centered" style={props.zIndex && { zIndex: props.zIndex }}>
        {props.children}
    </div>
);

export default Centered;
