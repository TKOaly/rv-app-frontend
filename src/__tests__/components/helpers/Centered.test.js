import Centered from '../../../components/helpers/Centered';
import React from 'react';
import renderer from 'react-test-renderer';

describe.only('<Centered />', () => {
    it('should match snapshot', () => {
        const cmpnt = renderer.create(<Centered />).toJSON();
        expect(cmpnt).toMatchSnapshot();
    });
});
