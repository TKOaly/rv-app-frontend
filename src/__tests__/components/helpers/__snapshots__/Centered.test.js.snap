// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Centered /> should match snapshot 1`] = `
<div
  className="centered"
/>
`;
